submarine-test/e2e/src/test/java/org/apache/submarine/integration/sidebarIT.java

```diff
@@ -66,7 +68,13 @@ public void sidebarNavigation() throws Exception {
     pollingWait(By.xpath("//span[contains(text(), \"Manager\")]"), MAX_BROWSER_TIMEOUT_SEC).click();
     pollingWait(By.xpath("//a[@href='/workbench/manager/user']"), MAX_BROWSER_TIMEOUT_SEC).click();
     SubmarineITUtils.sleep( 5000, true);
+
+    // Lazy-loading
+	  WebDriverWait wait = new WebDriverWait( driver, 15, 5000);
+	  pollingWait(By.xpath("//a[@href='/workbench/manager/user']"), MAX_BROWSER_TIMEOUT_SEC).click();  
+	  wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//span[@class='ant-breadcrumb-link ng-star-inserted']")));
```

Comment:
Wrong code indentation

Reply:
Thank for your review, @liuxunorg 
I have fixed the problem.